packages/plugin-payment-api/src/api/ErxesPayment.ts

```
@@ -24,51 +26,58 @@ class ErxesPayment {
+        this.minupay = new MinuPayAPI(payment.config, this.domain);
         break;
       case 'golomt':
-        this.golomt = new GolomtAPI(payment.config, domain);
+        this.golomt = new GolomtAPI(payment.config, this.domain);
         break;
       case 'stripe':
-        this.stripe = new StripeAPI(payment.config, domain);
+        this.stripe = new StripeAPI(payment.config, this.domain);
         break;
+      case 'khanbank':
+        this.khanbank = new KhanbankAPI(payment.config, subdomain || '');
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)